content/guides/user-guides/csi/ceph csi/_index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: CEPH CSI User Guide description: "A user guide for integrating SODA with CEPH CSI Driver via unified Plugin and
+hotpot project"
+weight: 30 disableToc: false tags: ["user guide", "csi", "kubernetes csi", "plug-n-play", "unified plugin"]
```

Comment:
disableToc and tags should be on newlines.

Reply:
@anvithks I have done the changes, please check.